step-templates/flyway-database-migrations.json

```diff
@@ -21,7 +21,7 @@
\n\t$dryRunOutputFile = Join-Path $(Get-Location) \"dryRunOutput\"\n    Write-Host \"Adding the argument dryRunOutput so Flyway will perform a dry run and not an actual migration.\"\n    $arguments += \"-dryRunOutput=`\"$dryRunOutputFile`\"\"\n}\n\n# Check to see if there's any additional arguments to add\nif (![string]::IsNullOrWhitespace($flywayAdditionalArguments))\n{\n\t# Split on space\n    $flywayAdditionalArgumentsArray = ($flywayAdditionalArguments.Split(\" \", [System.StringSplitOptions]::RemoveEmptyEntries))\n\n    # Loop through array\n    foreach ($newArgument in $flywayAdditionalArgumentsArray)\n    {\n    \t# Add the arguments\n    \t$arguments += $newArgument\n    }\n}\n\n# Display what's going to be run\nif (![string]::IsNullOrWhitespace($flywayUserPassword))\n{\n    $flywayDisplayArguments = $arguments.PSObject.Copy()\n    $arrayIndex = 0\n    for ($i = 0; $i -lt $flywayDisplayArguments.Count; $i++)\n    {\n        if ($null -ne $flywayDisplayArguments[$i])\n        {\n            if ($flywayDisplayArguments[$i].Contains($flywayUserPassword))\n            {\n                $flywayDisplayArguments[$i] = $flywayDisplayArguments[$i].Replace($flywayUserPassword, \"****\")\n            }\n        }\n    }\n\n    Write-Host \"Executing the following command: $flywayCmd $flywayDisplayArguments\"\n}\nelse\n{\n    Write-Host \"Executing the following command: $flywayCmd $arguments\"\n}\n\n# Attempt to find driver path for java\n$driverPath = (Get-ChildItem -Path (Get-ChildItem -Path $flywayCmd).Directory -Recurse | Where-Object {$_.PSIsContainer -eq $true -and $_.Name -eq \"drivers\"})\n\n# If found, add driver path to the PATH environment varaible\nif ($null -ne $driverPath)\n{\n\t$env:PATH += \"$([IO.Path]::PathSeparator)$($driverPath.FullName)\"\n}\n\n# Adjust call to flyway command based on OS\nif ($IsLinux)\n{\n    & bash $flywayCmd $arguments\n}\nelse\n{\n    & $flywayCmd $arguments\n}\n\n# Check exit code\nif ($lastExitCode -ne 0)\n{\n\t# Fail the step\n    Write-Error \"Execution of Flyway failed!\"\n}\n\n$currentDate = Get-Date\n$currentDateFormatted = $currentDate.ToString(\"yyyyMMdd_HHmmss\")\n\n# Check to see if the dry run variable has a value\nif (![string]::IsNullOrWhitespace($dryRunOutputFile))\n{     \n    $sqlDryRunFile = \"$($dryRunOutputFile).sql\"\n    $htmlDryRunFile = \"$($dryRunOutputFile).html\"\n    \n    if (Test-Path $sqlDryRunFile)\n    {\n    \tNew-OctopusArtifact -Path $sqlDryRunFile -Name \"$($flywayStepName)_$($flywayEnvironment)_$($currentDateFormatted)_dryRunOutput.sql\"\n    }\n    \n    if (Test-Path $htmlDryRunFile)\n    {\n    \tNew-OctopusArtifact -Path $htmlDryRunFile -Name \"$($flywayStepName)_$($flywayEnvironment)_$($currentDateFormatted)_dryRunOutput.html\"\n    }\n}\n\n$reportFile = Join-Path $(Get-Location) \"report.html\"\n    \nif (Test-Path $reportFile)\n{\n  \tNew-OctopusArtifact -Path $reportFile -Name \"$($flywayStepName)_$($flywayEnvironment)_$($currentDateFormatted)_report.html\"\n}",
```

Comment:
Would a `Write-Warning` be better in place of:

```ps
Write-Host "*****WARNING***** Use of the License Key has been deprecated by Redgate and will be removed in future versions, use the Personal Access Token method instead."
```

Reply:
Cool no worries. If it keeps it consistent thats better than changing all of the other entries imo